back/spec/acceptance/stats_ideas_spec.rb

```diff
@@ -48,23 +48,19 @@ def feedback_needed_filter_parameter(s)
     @timezone = AppConfiguration.instance.settings('core', 'timezone')
 
     @project1 = create(:project)
-    @project2 = create(:project)
     @project3 = create(:project)
```

Comment:
I know that's an annoying thing to ask, but maybe we can rename `project3` to `project2` now? It's not a biggie, but I'd be confused for a second looking at the spec and seeing `project1`, `project3`.